src/main/python/inverse_parser/_xml_node_children_getter.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Aniskov N.
+
+
+class _XmlNodeChildrenGetter:
+    @staticmethod
+    def get_children(xml_node, with_tag=None):
+        if with_tag:
+            return xml_node.findall(with_tag)
+
+        return xml_node.findall('*')
+
+    @staticmethod
+    def get_unique_child(xml_node, with_tag=None):
+        if with_tag:
+            res = xml_node.findall(with_tag)
+        else:
+            res = xml_node.findall('*')
```

Comment:
What is the difference between these lines and `get_children` function?

Reply:
You're right, thanks! I can replace that peace of code with simply `res = _XmlNodeChildrenGetter.get_children(xml_node, with_tag)`